lib/cdo/aws/cloud_formation.rb

```diff
@@ -15,27 +15,19 @@
 module AWS
   class CloudFormation
     # Hard-coded values for our CloudFormation template.
-    TEMPLATE = ENV['TEMPLATE'] || 'cloud_formation_adhoc_standalone.yml.erb'
+    TEMPLATE = ENV['TEMPLATE'] || raise('Stack template not provided in environment (TEMPLATE=[stack].yml.erb)')
     TEMPLATE_POLICY = TEMPLATE.split('.').tap {|s| s.first << '-policy'}.join('.')
     TEMP_BUCKET = ENV['TEMP_S3_BUCKET'] || 'cf-templates-p9nfb0gyyrpf-us-east-1'
 
     DOMAIN = ENV['DOMAIN'] || 'cdn-code.org'
 
-    # Lookup ACM certificate for ELB and CloudFront SSL.
-    ACM_REGION = 'us-east-1'.freeze
-    CERTIFICATE_ARN = Aws::ACM::Client.new(region: ACM_REGION).
-      list_certificates(certificate_statuses: ['ISSUED']).
-      certificate_summary_list.
-      find {|cert| cert.domain_name == "*.#{DOMAIN}" || cert.domain_name == DOMAIN}.
-      certificate_arn
```

Comment:
Refactored this constant to a method so this AWS API call is only made on-demand, not for every single stack creation/update/validation.